app/resources/api/v2/tag_group_adapter_type_resource.rb

```diff
@@ -2,15 +2,22 @@
 
 module Api
   module V2
-    # Provides a JSON API representation of a tag group AdapterType
-    # See: http://jsonapi-resources.com/ for JSONAPI::Resource documentation
+    # @todo This documentation does not yet include a detailed description of what this resource represents.
+    # @todo This documentation does not yet include detailed descriptions for relationships, attributes and filters.
+    # @todo This documentation does not yet include any example usage of the API via cURL or similar.
+    #
+    # @note Access this resource via the `/api/v2/tag_group_adapter_types/` endpoint.
+    #
+    # Provides a JSON:API representation of {TagGroup::AdapterType}.
+    #
+    # For more information about JSON:API see the [JSON:API Specifications](https://jsonapi.org/format/)
+    # or look at the [JSONAPI::Resources](http://jsonapi-resources.com/) package for Sequencescape's implementation
+    # of the JSON:API standard.
     class TagGroupAdapterTypeResource < BaseResource
-      model_name 'TagGroup::AdapterType', add_model_hint: false
+      model_name 'TagGroup::AdapterType', add_model_hint: true
```

Comment:
Model hint change intended? If so the default is true so it can be removed.

Reply:
Hmmm ... good question, I need to check this again.  I think the endpoint was not working until I changed this to `true`.  If it does need to be `true`, I will remove it as you suggested the default will be `true` anyway.  I'll check it in the morning.

Reply:
I remove the `add_model_hint: true` because, while it is needed, you don't need to do it explicitly like you said above.